packages/block-library/src/navigation/block.json

```
@@ -134,6 +134,9 @@
 		},
 		"interactivity": true
 	},
+	"selectors": {
+		"@currentItem": ".wp-block-navigation .current-item"
```

```suggestion
		"@currentItem": ".wp-block-navigation .current-menu-item"
```

For backwards compat reasons we'll want the selector to match what the block does currently.